packages/google_sign_in/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.2.1
+
+* Plugin can (once again) be used in apps that extend `FlutterActivity`
```

Comment:
You can remove the dependency on `com.android.support` from `build.gradle` now as well.